precheck.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+sbt clean scalafmt test:scalafmt coverage test coverageReport
```

Comment:
Please add formatting and running integration tests. From what I've checked, formatting can be done with: `it/Test/scalafmt`